<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_connect">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_connect">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="TextBrowser_log">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>701</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_loadtask">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>加载任务</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_runonce">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>执行一次</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_runduration">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>执行</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit_runduration">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_createtask">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>创建任务</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_runtimes">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>执行</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_runtimes">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_screenshot">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>截图</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
